src/test/isolation2/sql/standby_gxact_overflow.sql

```diff
@@ -0,0 +1,85 @@
+create or replace language plpythonu;
+create extension if not exists gp_inject_fault;
+
+--
+-- Test reproducing a gxact overflow on a standby
+-- The problem was caused by a race between a checkpoint and QD
+-- when they take a global transaction snapshot (issue #9207)
+--
+-- Before current fix gxact stucked in shmCommittedGxactArray and
+-- after try #2 we got an overflow on a standby.
+--
+-- start_ignore
+1: create table t (a int);
```

Comment:
Relation with the same name already exists (was created and wasn't drop) in https://github.com/arenadata/gpdb/blob/35827cabc11f70da2db583b26da8f647ae04e0d4/src/test/isolation2/sql/segwalrep/failover_with_many_records.sql#L12

Reply:
Fixed